src/test/enhetstester/kotlin/no/nav/familie/ba/sak/kjerne/vilkårsvurdering/VilkårsvurderingValideringTest.kt

```diff
@@ -251,6 +263,272 @@ class VilkårsvurderingValideringTest {
         }
     }
 
+    @Nested
+    inner class ValiderAtDetIkkeFinnesDeltBostedForBarnSomIkkeBorMedSøkerITilleggssone {
```

Comment:
Burde denne hete `ValiderAtDetIkkeFinnesDeltBostedForBarnSomIkkeBorMedSøkerIFinnmark` da det ser ut til å være den metoden som blir kalt i "act" steget

```suggestion
    inner class ValiderAtDetIkkeFinnesDeltBostedForBarnSomIkkeBorMedSøkerIFinnmark {
```